english/staypuft/place-hat/how-did-you-know/father.md

```diff
@@ -0,0 +1,12 @@
+"No. No. That's not true. That's impossible" I reply.
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.